sdk/metric/aggregator/test/test.go

```diff
@@ -66,21 +63,6 @@ func RunProfiles(t *testing.T, f func(*testing.T, Profile)) {
 	}
 }
 
-// Ensure local struct alignment prior to running tests.
-func TestMain(m *testing.M) {
```

Comment:
I don't think should have been removed. `Numbers.numbers` still needs to be 64-bit aligned, right?

Reply:
Fixed.

I was thinking about it differently, but I think you're right. I do not expect atomic access to the entries in the array, but the contract of a `metric.Number` is that it must be aligned because it exposes `Atomic` methods.